patterns/template-home-with-sidebar-news-blog.php

```diff
@@ -85,7 +85,7 @@
 						<div class="wp-block-group has-small-font-size">
 							<!-- wp:post-terms {"term":"category","style":{"typography":{"textTransform":"uppercase","letterSpacing":"1.4px"}}} /-->
 							<!-- wp:paragraph -->
-							<p><?php echo esc_html_x( '·', 'Separator between date and categories', 'twentytwentyfive' ); ?></p>
```

Comment:
Without the period at the end of the context, this created a separate string.
https://translate.wordpress.org/projects/wp-themes/twentytwentyfive/fr/default/?filters%5Bterm%5D=Separator+between+date+and+categories